packages/bpk-component-popover/src/bpk-popover.scss

```diff
@@ -18,21 +18,39 @@
 
```

Comment:
Let's rename this file as per [this decision](https://github.com/Skyscanner/backpack/blob/master/decisions/component-scss-filenames.md).

Reply:
👍